sdk/runtime_tests/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 The Kubeflow Authors
```

Comment:
Do we really need this file in the test folder?

Reply:
No. Good catch. Removed.